NDTensors/src/abstractarray/fill.jl

```diff
@@ -1,13 +1,9 @@
-function generic_randn(arraytype::Type{<:AbstractArray}, dim::Integer=0)
+function generic_randn(arraytype::Type{<:AbstractArray}, dim::Integer=0; rng = Random.default_rng())
   arraytype_specified = set_unspecified_parameters(
     leaf_parenttype(arraytype), DefaultParameters()
   )
   data = similar(arraytype_specified, dim)
-  ElT = eltype(data)
-  for i in 1:length(data)
-    data[i] = randn(ElT)
-  end
-  return data
+  NDTensors.randn!(rng, data)
```

Comment:
Good idea. Can we do something similar with `generic_zeros` and call `zeros!`?

Reply:
Also, I don't think we need `NDTensors.` here, `randn!` is an exported `Base` function.

Reply:
Yes I was able to find a more generic zeros initializer